pkg/networks/commands.go

```diff
@@ -74,6 +112,21 @@ func (config *NetworksConfig) StartCmd(name, daemon string) string {
 				nw.Gateway, nw.DHCPEnd, nw.NetMask)
 		}
 		cmd += " " + config.VDESock(name)
+	case SocketVMNet:
+		nw := config.Networks[name]
+		if config.Paths.SocketVMNet == "" {
+			panic("fuck")
```

Comment:
Probably should not have profanity in the software...

Reply:
Ah, just forgot to remove the debug print...

Reply:
fixed